iptables/iptables.go

```diff
@@ -33,8 +34,14 @@ func AddRule(appPort, metadataAddress, hostInterface, hostIP string) error {
 	return nil
 }
 
-// CheckInterfaceExists - validates the interface passed exists for the given system
+// CheckInterfaceExists - validates the interface passed exists for the given system, ignores calico networks
```

Comment:
Nit: it could be other networks as well, it's really about ignoring wildcard characters so i'd make this comment and the one below more generic.